### PR TITLE
wait-for helper's count value is null by default.

### LINKED ISSUE #312: Documentation of the `count` option of the `waitFor` helper is wrong
The documentation suggests that the default is `1`, but in fact it is `null` which basically means "one or more" instead of "exactly one".

### PR DESCRIPTION
**Pull request by sly7-7**
Closes #312

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -41,10 +41,10 @@
     -   [setupRenderingContext](#setuprenderingcontext)
     -   [teardownRenderingContext](#teardownrenderingcontext)
     -   [setApplication](#setapplication)
-    -   [getApplication](#getapplication)
     -   [setupApplicationContext](#setupapplicationcontext)
     -   [teardownApplicationContext](#teardownapplicationcontext)
     -   [validateErrorHandler](#validateerrorhandler)
+-   [getApplication](#getapplication)
```
Comment (rwjblue) [inline API.md]: I assume this was updated due to running `yarn docs`?  We need to tweak the configuration to show `getApplication` under the `Test Framework APIs` section.  I think all we need to do is add it to this list https://github.com/emberjs/ember-test-helpers/blob/184bb747a2d08f5af86d545e2f4a79db96301f51/documentation.yml#L42

Comment (sly7-7) [reply API.md]: Yep, it was a updated as a side effect. Do you want a separate PR ?

Comment (Turbo87) [reply API.md]: yeah, I guess this PR should focus on the `waitFor` thing only

**Comment by rwjblue** [comment]
Thank you!

**Pull request merged**